tests/e2e-tests/specs/admin/shipping-settings.test.js

```diff
@@ -0,0 +1,348 @@
+/**
+ * Internal dependencies
+ */
+import { StoreOwnerFlow } from "../../utils/flows";
+import { AccountWithNoCreditCard, AccountWithOneCreditCard, AccountWithTwoCreditCard, AccountWithTwoCreditCardAndNoDefault } from "../../fixtures/account_settings";
+
+// Click save and wait until it's saved.
+const saveAndWait = async () => {
+    await expect( page ).toClick( '.button.is-primary', { text: 'Save changes' } );
+    await page.waitForSelector('[class="button is-primary"]');
+};
+
+/**
+ * This function will wait for a button with any CSS selector + text value.
+ *
+ * @param {string} selector CSS selector
+ * @param {string} text The text value of the element we want to search for. ie. button's value, div's innertext.
+ */
+const waitForSelectorAndText = async (selector, text) => {
```

Comment:
We can't use `waitForSelector` with a `text` option. That's not available in the Puppeteer API. In order to do something similar to what `expect-puppeteer` does with the `toMatchElement` API (https://github.com/smooth-code/jest-puppeteer/tree/master/packages/expect-puppeteer#expectinstancetomatchelementselector-options), which support a `text` option, i created this function. 


await page.waitForSelector( '.address-step__suggestion-title', { text: 'Address entered' } );`

Reply:
Love those helpers! I would refactor this and probably move them to a separate file, what do you think?

Reply:
I moved `waitForSelectorAndText` into `utils/index.

I didn't move `saveAndWait` since it is clicking a "Save changes" button. I think "Save changes" is more specific to just this account settings page so I didn't move it.